tlf-disposition.Rmd

```diff
@@ -134,7 +138,7 @@ tbl_disp <- bind_rows(n_rand, n_complete, n_disc, n_reason) %>%
 tbl_disp
 ```
 
-We start to define the format of the output.
+In the below code, formatting of the final table is defined. 
 We highlighted items that is not discussed in the previous discussion.
```

Comment:
Items that were not discussed in the previous sections, are highlighted below.

Reply:
```suggestion
Items that were not discussed in the previous sections, are highlighted below.
```